modules/node-certificates.adoc

```diff
@@ -0,0 +1,29 @@
+
+// Module included in the following assemblies:
+//
+// * authentication/understanding-certificates.adoc
+
+[id="node-certificates_{context}"]
+= Node certificates
```

Comment:
Should we talk about the CA's and the validity of these? How long they are valid for; how to rotate them, etc?

@rphillips @sjenning can either of you comment on this?

Reply:
@rphillips @sjenning Can you please comment?

Reply:
@ahardin-rh this is correct. We might want to add that once the cluster is installed the Node certificates are auto-rotated.